docs/docs/development/usb-logging.mdx

```diff
@@ -89,27 +96,31 @@ From there, you should see the various log messages from ZMK and Zephyr, dependi
 
 Standard boards such as the nice!nano and Seeeduino XIAO family have the necessary configuration for logging already added, however if you are developing your own standalone board you may wish to add the ability to use USB logging in the future.
 
-To add USB logging to a board you need to define the USB CDC ACM device that the serial output gets piped to, as well as adding the console in the `chosen` node inside `<board>.dts`.
+To do so, you need to follow the upstream Zephyr [`cdc-acm-console` snippet requirements](https://docs.zephyrproject.org/3.6.0/snippets/cdc-acm-console/README.html#requirements) steps.
 
-Inside the USB device (`&usbd`), add the CDC ACM node:
+Usually, this just requires ensuring that the USB node has been tagged with the `zephyr_udc0` label, e.g.
 
-```dts
-&usbd {
+```
+zephyr_udc0: &usbd {
     status = "okay";
-    cdc_acm_uart: cdc_acm_uart {
-        compatible = "zephyr,cdc-acm-uart";
-    };
 };
 ```
 
-Then you can add the `zephyr,console` binding in the `chosen` node:
+## Enabling Logging On Older Boards
 
-```dts
-/ {
-    chosen {
-        ...
-        zephyr,console = &cdc_acm_uart;
-    };
-    ...
-};
+Previously, enabling logging required editing Kconfig files to enable logging. If for whatever reason a build fails when using
+a custom board definiton that does not support the new `zmk-usb-logging` snippet, you can use the following steps:
+
+The `CONFIG_ZMK_USB_LOGGING` Kconfig enables USB logging. This can be set at the keyboard level, typically in the `config/<your_keyboard>.conf`
+file if you are using a [user config repository](user-setup.mdx). It can also be enabled at the ZMK level using the `app/prj.conf` file, or other
+search locations described in the [configuration overview](config/index.md#config-file-locations).
```

Comment:
```suggestion
Previously, enabling logging required setting the `CONFIG_ZMK_USB_LOGGING` Kconfig symbol. If for whatever reason
a custom board definition does not support the new `zmk-usb-logging` snippet, you can try setting this symbol at the keyboard level, typically in the `config/<your_keyboard>.conf`
file if you are using a [user config repository](user-setup.mdx). It can also be enabled at the ZMK level using the `app/prj.conf` file, or other
search locations described in the [configuration overview](config/index.md#config-file-locations).
```
The "build fails for whatever reason" part seemed conflicting with the "not supporting the snippet," so I removed it. Let me know if there was nuance I was missing there.